Author: Stefano Rivera <stefano@rivera.za.net>
Description: We use a private Python module directory, patch it into sys.path
 at the start of the binaries.
Forwarded: not-needed
Last-Update: 2010-06-27

---
 disk-manager.in |   10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

--- disk-manager.orig/disk-manager.in
+++ disk-manager/disk-manager.in
@@ -29,15 +29,7 @@ from gettext import gettext as _
 
 import gtk
 
-# Hack to make sure that path is set correctly when installation is in /usr/local
-if "/usr/local" in sys.argv[0] :
-    for path in sys.path[:] :
-        if "/usr/" in path :
-            local_path = path.replace("/usr", "/usr/local")
-            if not local_path in sys.path :
-                sys.path.append(path.replace("/usr", "/usr/local"))
-                
-print sys.path
+sys.path.insert(0, '/usr/share/disk-manager')
 
 from DiskManager.DiskManager import *
 from DiskManager.Config import Config
